docs/manual/developer/04_styleguide.md

```diff
@@ -0,0 +1,100 @@
+# Styleguide
+
+## All files
+* Use UNIX style line endings
+* One blank line at the end of the file
+
+## Python
+* All Python files should follow [PEP 8](https://www.python.org/dev/peps/pep-0008/). We use
+[PEP 8 Speaks](https://pep8speaks.com/). We do make one change from PEP 8, our maximum
+line length is 99 characters. 
+* Methods should be defined before they called.
+
```

Comment:
Might be nice to clarify here that we prefer Py2+Py3 compatible files, though when necessary, utilities can be Python3-only (see e.g. https://github.com/ComplianceAsCode/content/pull/7454#discussion_r697872005)

Reply:
Great point! We discussed this internally couple of weeks ago and we prefer switching to Python 3. Python 2 was needed due to RHEL7 and similar, but starting from RHEL 7.7 there is Python 3.6 in RHEL 7, so  Python 2 isn't needed anymore.

Reply:
Not so fast, we haven't made the switch yet, so build-related and test-related files should be compatible for the time being. I hope that we will be able to migrate to 3.6 this calendar year.

Reply:
I will keep `The files in the build system shall be Python 2.7 and Python 3 compatible` in the Python section then.